### PR TITLE
Added a setting for the verse for the 3rd antiphon -- NativityTheotokos

### PR DESCRIPTION
**Pull request by JedBartlett**
We've always just straight-chanted this (or intoned it to match the others), but since we're going for consistency....

### CODE REVIEW

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  g4 a b b b b b g a2 a4 a g2 \bar "||"
+  g4 a b g c2. b2 r4 \bar "|"
+  g4 a b g a2 a2 g2 \bar "|."
+}
+
+alto = \relative g' {
+  \global
+  % Verse 1
+  g4 g g g g g g g2 g4 g4 r4
+  g4 g g g  g g g g fis2 fis d2
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: fis2 fis should be fis2 fis4 fis

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  g4 g g g g2. g2 r4
+  g4 g g g d1 d2
+}
+
+tenor = \relative c' {
+  \global
+  % Verse 1
+  b4 b d d d d d e2 e4 d4 r4
+  b4 d d d d d d b c2 c4 c b2
+  b4 d d d e2 c4 d2 r4
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: e2 c4 should be e2 (c4)

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  g4 a b g a2 a2 g2 \bar "|."
+}
+
+alto = \relative g' {
+  \global
+  % Verse 1
+  g4 g g g g g g g2 g4 g4 r4
+  g4 g g g  g g g g fis2 fis d2
+  g4 g g g g2. g2 r4
+  g4 g g g d1 d2
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: d1 d2 should be d2 d d

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+\defineBarLine "invisible" #'("" "" "")
+global = {
+  \time 1/1 % Not used, Time_signature_engraver is removed from layout
+  \key g \major
+  \set Timing.defaultBarType = "invisible" %% Only put bar lines where I say
+}
+
+verseOne = \lyricmode {
+  Here will I dwell, for I have cho -- sen her.
+  The Most High hath hall -- ow -- ed His tab -- er -- na -- cle.
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: hal -- lowed, I know we often split this word into 3 syllables liturgically, but here the duple meter makes "-ed" sound accented, so 2 syllables would sound slightly better. Also it just feels better to me to have it as 2 syllables when used as a verb.
tab -- er -- nac -- le.

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  Ho -- ly is Thy tem -- ple,
+  won -- der -- ful in righ -- teous -- ness.
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: right -- eous -- ness.

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+soprano = \relative g' {
+  \global
+  % Verse 1
+  g4 g a b b a b c2 c4 b4 r4 \bar "|"
+  g4 a b b b b b g a2 a4 a g2 \bar "||"
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: Combine "-low-" and "-ed" so change
g4 a b b b b b g
to
g4 a b b b b g

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly (hunk 1 above)
Comment (ghost) [reply MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: Also remove a g for "-lowed"

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+  g4 g g g  g g g g fis2 fis d2
+  g4 g g g g2. g2 r4
+  g4 g g g d1 d2
+}
+
+tenor = \relative c' {
+  \global
+  % Verse 1
+  b4 b d d d d d e2 e4 d4 r4
+  b4 d d d d d d b c2 c4 c b2
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: Remove a d for "-lowed"

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+  b4 d d d e2 c4 d2 r4
+  b4 d d d c2 c2 b2
+}
+
+bass = \relative c {
+  \global
+  % Verse 1
+  g'4 g g g g g g c,2 e4 g4 r4
+  g4 g g g g g g g d2 d4 d g2
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: Remove a g for "-lowed"

**Review by ghost — CHANGES_REQUESTED** [review]
I really like what you did with this one! Just found a few typos.

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly (hunk 1 above)
Comment (JedBartlett) [reply MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: What do you mean remove a g ?

Comment (JedBartlett) [reply MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: image
?

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly (hunk 4 above)
Comment (JedBartlett) [reply MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: What lexical phoneme dictionary are you using?  we should use the same one.  I checked the dictionary splitting with: https://www.merriam-webster.com/dictionary/tabernacle

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly (hunk 6 above)
Comment (JedBartlett) [reply MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: Ok, i guess this is what you meant above.

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly (hunk 1 above)
Comment (ghost) [reply MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: To match the change in the lyrics with hal-lowed as 2 syllables

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly (hunk 4 above)
Comment (ghost) [reply MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: Yeah sorry, I should have realized that some dictionaries disagree on syllable splitting. Let's use https://www.howmanysyllables.com/ going forward, as it's intended for that purpose.

Comment (ghost) [reply MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: This one also looks promising: http://juiciobrennan.com/hyphenator/

Comment (ghost) [reply MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: Also relevant regarding latex: https://tex.stackexchange.com/questions/133859/using-tex-to-split-words-into-syllables

File: MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly [hunk 9]
```diff
@@ -23,16 +23,16 @@ global = {
 
 verseOne = \lyricmode {
   Here will I dwell, for I have cho -- sen her.
-  The Most High hath hall -- ow -- ed His tab -- er -- na -- cle.
+  The Most High hath hall -- owed His tab -- er -- nac -- le.
```
Comment (ghost) [inline MusicTexts/Menaion/09-08-NativityOfTheTheotokos/NativityTheotokos-Antiphon3-rdrG.ly]: hal -- lowed

**Review by ghost — APPROVED** [review]

**Pull request merged**